roles/baseline/defaults/main.yml

```diff
@@ -30,7 +30,9 @@ INGRESS_NGINX_NAME: ingress-nginx
 INGRESS_NGINX_NAMESPACE: ingress-nginx
 INGRESS_NGINX_CHART_NAME: ingress-nginx
 INGRESS_NGINX_CHART_URL: https://kubernetes.github.io/ingress-nginx
-INGRESS_NGINX_CHART_VERSION: 3.20.1
+INGRESS_NGINX_CHART_VERSION_K8S_21_AND_BELOW: 3.20.1
+INGRESS_NGINX_CHART_VERSION_K8S_22_AND_ABOVE: 4.0.13
+INGRESS_NGINX_CHART_VERSION: UNSPECIFIED
```

Comment:
This format changes the entire templating for helm charts. I would not approve this.

Reply:
Updated in latest push for review.